torchaudio/datasets/librimix.py

```diff
@@ -55,45 +55,62 @@ def __init__(
         if mode not in ["max", "min"]:
             raise ValueError(f'Expect ``mode`` to be one in ["min", "max"]. Found {mode}.')
         if sample_rate == 8000:
-            self.root = self.root / "wav8k" / mode / subset
+            mix_dir = self.root / "wav8k" / mode / subset
         elif sample_rate == 16000:
-            self.root = self.root / "wav16k" / mode / subset
+            mix_dir = self.root / "wav16k" / mode / subset
         else:
             raise ValueError(f"Unsupported sample rate. Found {sample_rate}.")
         self.sample_rate = sample_rate
         self.task = task
-        self.mix_dir = (self.root / _TASKS_TO_MIXTURE[task]).resolve()
+
+        self.mix_dir = mix_dir / _TASKS_TO_MIXTURE[task]
```

Comment:
@nateanl do you know why `.resolve()` was used here and if it's ok to remove? including it causes some tests to fail because it sometimes adds `\private\` to the filepath and messes up with the relative path for finding the metadata filepath.

Reply:
`.resolve()` is to convert relative path like `..` to absolute path. This is based on wsj0-2mix dataset under examples folder, so @mthrok may know the reason behind it. I feel it can be removed, as other datasets don't rely on `resolve` method.